### PR TITLE
doc: Offer replication_factor=3 as the default in the examples

### PR DESCRIPTION
**Pull request by KrzaQ**
The goal is to make the available defaults safe for future use, as they are often taken from existing config files or documentation verbatim.

Referenced issue: #14290

@annastuchlik Please give it a look if there are other places that require changes to NetworkTopologyStrategy or RF=3. I think I caught them all.

### CODE REVIEW

**Review by tzach — APPROVED** [review]
LGTM

**Comment by bhalevy** [comment]
Why the PR is marked as Draft?

**Comment by ptrsmrn** [comment]
Is this PR and [the previous one](https://github.com/scylladb/scylladb/pull/15856) eligible for the 5.4 backport? @annastuchlik ? If so, @KrzaQ please add 'Requires-backport-to-5.4' label to both PRs, and please ping Avi on the older PR that it also needs to be backported.

File: docs/operating-scylla/admin-tools/cassandra-stress.rst [hunk 1]
```diff
@@ -94,7 +94,7 @@ CQL for the keyspace. Optional if the keyspace already exists:
 .. code-block:: cql
 
    keyspace_definition: |
-   CREATE KEYSPACE stresscql WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+   CREATE KEYSPACE stresscql WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 3};
```
Comment (ptrsmrn) [inline docs/operating-scylla/admin-tools/cassandra-stress.rst:97]: As this file relates to not only Scylla, but Cassandra as well, and also to cassandra-stress, which I'm not familiar with, I'd double check if that very line isn't actually used in our tests somewhere. Perhaps it's a line of code extracted from some function from the framework? (In which case we'd need to change it as well?)

File: docs/operating-scylla/security/rbac-usecase.rst [hunk 2]
```diff
@@ -87,9 +87,9 @@ If you proceed with the following example without authorization, you will see th
 
 .. code-block:: cql
 
-   CREATE KEYSPACE IF NOT EXISTS customer WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', 'replication_factor' : 1 };
+   CREATE KEYSPACE IF NOT EXISTS customer WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', 'replication_factor' : 3 };
```
Comment (ptrsmrn) [inline docs/operating-scylla/security/rbac-usecase.rst:90]: A quick comment: i'm not sure increasing RF doesn't impact the behavior of the test (?)

**Review by ptrsmrn — COMMENTED** [review]
In general, changes look good, although I'd first resolved the comments I placed.

File: docs/operating-scylla/admin-tools/cassandra-stress.rst (hunk 1 above)
Comment (ptrsmrn) [reply docs/operating-scylla/admin-tools/cassandra-stress.rst:97]: @fruch is it something you're familiar with?

File: docs/operating-scylla/security/rbac-usecase.rst (hunk 2 above)
Comment (ptrsmrn) [reply docs/operating-scylla/security/rbac-usecase.rst:90]: I'm taking back what I said, this looks OK.

File: docs/operating-scylla/admin-tools/cassandra-stress.rst (hunk 1 above)
Comment (ptrsmrn) [reply docs/operating-scylla/admin-tools/cassandra-stress.rst:97]: @fruch could you please comment? If not, we'll resolve this comment and proceed. (We can always fix that later).

Comment (ptrsmrn) [reply docs/operating-scylla/admin-tools/cassandra-stress.rst:97]: @KrzaQ this specific line describes what's the default test behaviour and as @annastuchlik pointed out, below in the FAQ section there's an explanation what to do in order to run tests with NetworkTopology.
For this reason I think the change in this specific line should be reverted.

Comment (KrzaQ) [reply docs/operating-scylla/admin-tools/cassandra-stress.rst]: done

**Review by ptrsmrn — APPROVED** [review]
@KrzaQ you can make it ready for review, for me the changes look legit.

**Comment by ptrsmrn** [comment]
@annastuchlik do you intend to look again at this PR or should we just merge it? We already have 2 approves.

**Comment by ptrsmrn** [comment]
@nyh could you please merge? This is a doc change removing/replacing SimpleStrategy to NetworkTopology, and it's been already approved by me, Tzach and Anna (https://github.com/scylladb/scylladb/pull/15947#pullrequestreview-1717843936.

**Comment by ptrsmrn** [comment]
> @nyh could you please merge? This is a doc change removing/replacing SimpleStrategy to NetworkTopology, and it's been already approved by me, Tzach and Anna ([#15947 (review)](https://github.com/scylladb/scylladb/pull/15947#pullrequestreview-1717843936).

@nyh @denesb can somebody please merge this PR?

**Comment by mykaul** [comment]
ping @scylladb/scylla-maint  for merge.

**Pull request closed without merging**